recipes/tobac/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tobac" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/climate-processes/{{name}}/archive/v{{version}}.tar.gz
+  sha256: 74a47a181e9a4b4df7f993165ac31aeb870efcdd8c3ffe6b1da4c963f80eb997
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-image
+    - pandas
+    - pytables
+    - matplotlib 
+    - iris 
+    - xarray 
+    - cf_units
+    - cartopy
+    - trackpy
+
+test:
+  imports:
+    - tobac
+    - tobac.testing
+  source_files:
+    - tobac/tests/test_import.py
+    - tobac/tests/test_sample_data.py
+  requires:
+    - pytest
+  commands:
+    - pytest tobac/tests
+about:
+  home: http://github.com/simplejson/simplejson
+  license:  BSD-3-Clause 
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Tracking and Object-Based Analysis of Clouds'
+
+  description:  tobac is a flexible frameork for cloud tracking of different types of datasets
+  doc_url: http://tobac.readthedocs.io/
+  dev_url: https://github.com/climate-processes/tobac
+
+extra:
+  recipe-maintainers:
+    - mheikenfeld
+    - w-k-jones
```

Comment:
@w-k-jones do you agree to be a maintainer of this recipe as well?

Reply:
He has agreed to me in a conversation yesterday that he'll be happy to act as a maintainer.

Reply:
Just confirming that I'm happy to be a maintainer of this recipe as well